docs/releases/1.11.txt

```diff
@@ -488,6 +488,10 @@ Miscellaneous
   ``migrate`` command. Use the new :djadmin:`remove_stale_contenttypes` command
   instead.
 
+* ETags are now parsed according to the updated :rfc:`7232` Conditional Requests
```

Comment:
Perhaps a bit simpler:
```
* ETags are now parsed according to the :rfc:`7232` Conditional Requests
  specification rather than the syntax from :rfc:`2616`.
```